### DISCUSSION TITLE
WebRAVE Launched to production!! (and URL Change)

### DISCUSSION THREAD

**Discussion by MattReimer** [Riverscapes/RiverscapesXML#367] [RAVE: Symbology/BusinessLogic] [answered]
Hey @Riverscapes/riverscape-project-curators 

Last Sunday we deployed webRAVE to the production server and built the tiles. As a result you can (and should) base all your work on a new set of urls. 

### From this point forward:

* Warehouse links will be <https://data.riverscapes.xyz>
* webRAVE links will be `https://webrave.riverscapes.xyz/...`

The staging address (the one that begins with `northarrowresearchlabs.github.io/....`) is now going back to being our testing server that I reserve the right to break and try things on so please update any bookmarks you might have. I'm going to remove some access to it for now just so nobody gets confused while we transition.

Thanks to everyone for your patience, diligence and a flurry of pull requests. We're definitely not done but we're at a place where people are going to start seeing our hard work and that's pretty exciting!

**Comment by CHafen** [comment]
Hey @MattReimer I'm having trouble accessing the warehouse and webRAVE. When I try to access the warehouse it tells me no access for things and for webRAVE it doesn't want to load (it just stays on the screen from the image below). Any suggestions?
![webrave](https://user-images.githubusercontent.com/39837888/140407375-bc03169b-8fb1-45e3-95c0-bc4e895694f4.png)

**Reply by MattReimer** [reply]
No but I'm on it

**Reply by MattReimer** [reply]
Ok. finally back at my desk. @Cashe93 that url doesn't actually lead anywhere. Webrave needs to be pointed at a specific program and project.

Granted I should have a better message and I'm making a note to improve that but if you got to https://data.riverscapes.xyz/#/ I'm guessing you'll see a warehouse (I do, let me know if you don't)

**Reply by CHafen** [reply]
Ok, thanks @MattReimer that makes sense. I do see a warehouse when I go to  https://data.riverscapes.xyz/#/ but I don't have access to any of the programs
![warehouse](https://user-images.githubusercontent.com/39837888/140525202-e0e08278-24c1-4c5e-9e9a-d1bb3e38998b.png)

**Reply by joewheaton** [reply]
@CHafen I just added you to the ones I had administrative access to. Try now.

**Reply by CHafen** [reply]
Thanks @joewheaton looks like I have access to things now

**Comment by joewheaton** [comment, marked as answer]
This is implemented.
